README.asciidoc

```diff
@@ -42,9 +42,9 @@ On top of that, to build Debian packages and Neo4j Desktop:
 
 == Building Neo4j ==
 
-Before you start running the unit and integration tests in the Neo4j Maven project on a Linux-like system, you should ensure your limit on open files is set to a reasonable value. You can test it with `ulimit -n`. We recommend you have a limit of at least 40K. 
+Before you start running the unit and integration tests in the Neo4j Maven project on a Linux-like system, you should ensure your limit on open files is set to a reasonable value. You can test it with `ulimit -n`. We recommend you have a limit of at least 40K.
 
-* A plain `mvn clean install` will only build the individual jar files. 
+* A plain `mvn clean install` will only build the individual jar files.
 * Test execution is, of course, part of the build.
 * In case you just want the jars, without running tests, this is for you: `mvn clean install -DskipTests`.
 * To build product packages, do `export PATH="bin:$PATH" && make clean all` in the packaging directory after building artifacts with Maven
```

Comment:
while you still can do that, now there is no need in ``The license header check can be skipped by appending the following to the command line: `-Dlicense.skip=true`.`` step on Windows build in since latest version can handle licence files on windows as well

Reply:
addressed below. Many thanks